README.md

```diff
@@ -1,3 +1,56 @@
-# bonsai
+#  (Work-in-progress) Bonsai
 
-(Work-in-progress) Future collection of JAX model implementations.
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
+
+A collection of State-of-the-Art models implemented using JAX [NNX](https://flax.readthedocs.io/en/v0.8.3/experimental/nnx/index.html). This repository serves as a curated list of well-documented and easy-to-understand JAX implementations of common machine learning models.
```

Comment:
Do we wanna say State of the Art? Like what does that mean? Maybe just remove "A collection of models..."